conf/cassandra.yaml

```diff
@@ -566,6 +566,14 @@ commitlog_segment_size: 32MiB
 #
 # flush_compression: fast
 
+# Defines the compression params used to read/write the sstables.  If undefined the default is LZ4 with 16kb chunk
+# length and min_compress_ratio of 0.0 (see CompressionParams.DEFAULT_MIN_COMPRESS_RATIO).
+# This is the default options for sstable compression, the (schema) table-level settings will override this option.
+#sstable_compressor:
+#  class: "org.apache.cassandra.io.compress.LZ4Compressor"
```

Comment:
```suggestion
#  class_name: "LZ4Compressor"
```

here, and in table options, it would be nice to have some aliases to known compressors: none, lz4, snappy, deflate, zstd.

Reply:
I disagree. We are using here the very same approach where we can plug-in things. It is well-known pattern that these things are referenced by FQCN (or just simple class name and package will be resolved automatically).

Reply:
what's the harm is providing aliases to the simple class names ? 

I see benefit and simplicity in permitting the user to define any of the following (to the same effect)
```
class: "lz4"
class: "LZ4Compressor"
class: "org.apache.cassandra.io.compress.LZ4Compressor"
```

Reply:
ah right ... as long as we are not introducing some "type" field or similar ... that might work. Let's explore that. But anyway, "lz4" is not a _class_ to be exact.